<h1>
 Title: Testing addconflicts
</h1>
<h3>
 Author: Anonymous
</h3>
<h3>
 Date: 2021-05-15T10:43:36.671918+10:00
</h3>
<h3>
 Category: Assignments
</h3>
<h3>
 Vote Count: 0
</h3>
<p>
 I am getting this error when I test my code, and I am not sure what it means. Is this an error with the input? I am using schedule_tests.py .
</p>
<figure>
 <image height="296.51346499102334" src="https://static.us.edusercontent.com/files/v7CEJvgByebCJSCSvdFjlhGm" width="658"/>
</figure>
<h3>
 ----------- REPLIES -----------
</h3>
<div style="text-indent: 2em;">
 <h3>
  Author: Troy Moslemi (he/him) (student)
 </h3>
 <h3>
  Vote Count: 0
 </h3>
 <p>
  Hi,
 </p>
 <p>
  You are calling conflictsWith in another function, right? The arguments you need to call conflictsWith are course objects. I believe you are calling conflictsWith with strings.
 </p>
 <p>
  Best,
 </p>
 <p>
  Troy
 </p>
 <h3>
  ------------------------------------
 </h3>
</div>